Cup_of_join.py

```diff
@@ -0,0 +1,16 @@
+def join(*lists, sep='-'):
+    '''
+    get unlimited number of lists and combined them into one list, seperated
+    with the sep sign
+    :param lists: lists wanted to be combined
+    :param sep: separator between the lists
+    :return: the combined list
+    '''
+    if not lists:
```

Comment:
https://stackoverflow.com/questions/1839289/why-should-functions-always-return-the-same-type